packaging/helm-charts/helm3/strimzi-drain-cleaner/values.yaml

```diff
@@ -70,6 +70,8 @@ podLabels: {}
   # Specifies optional custom labels for Pod.
   # If you want to add custom labels, add them below and remove curly braces above.
 
+annotations: {}
```

Comment:
You are adding a field for pod annotations -> I think that is fine if there is some value for it. But it should be clearly named as `podAnnotations`.

Reply:
Updated in latest commit 👍